src/common/schema/application.js

```diff
@@ -390,11 +390,11 @@ module.exports = {
     'dischargeType'
   ],
   dependencies: {
-    veteranGrossIncome: ['understandsFinancialDisclosure'],
-    veteranNetIncome: ['understandsFinancialDisclosure'],
-    veteranOtherIncome: ['understandsFinancialDisclosure'],
-    spouseGrossIncome: ['understandsFinancialDisclosure'],
-    spouseNetIncome: ['understandsFinancialDisclosure'],
-    spouseOtherIncome: ['understandsFinancialDisclosure']
+    veteranGrossIncome: { properties: { understandsFinancialDisclosure: { 'enum': [true] } } },
```

Comment:
That rings true to me. @alexose If you can confirm you tested this against a few docs, 👍